models/comments-models.js

```diff
@@ -0,0 +1,11 @@
+const db = require("../db/connection");
+
+exports.fetchComments = (article_id) => {
+  const query = `SELECT * FROM comments WHERE article_id =$1 ORDER by created_at DESC`;
+  return db.query(query, [article_id]).then(({ rows, rowCount }) => {
+    if (rowCount === 0) {
+      return Promise.reject({ status: 404, msg: "Article not found" });
+    }
```

Comment:
This error relates to an article not found. But you don't need it as you are checking if the article exists in the controller. If it doesn't, that model will throw the error